code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -8,7 +8,8 @@ GLOBAL_LIST_INIT(oilfry_blacklisted_items, typecacheof(list(
 	/obj/item/reagent_containers/food/condiment,
 	/obj/item/storage,
 	/obj/item/small_delivery,
-	/obj/item/his_grace)))
+	/obj/item/his_grace,
+	/obj/item/bodybag/bluespace)))
```

Comment:
I believe this could easily be circumverted this in several ways, one of them being inserting the bodybag into a cake of cheese wheel and then deefrying the food item instead, so I doubt this will fix the ""issue"" of deleting mobs in a fanciful way.

Reply:
We block storage items across the board, the bigger thing is more blocking the ability to put faux storage items in the deep fryer such as bodybags.